### PR TITLE
[IA-1335] Update wb-libs for IAM fixes

### PR DESCRIPTION
**Pull request by rtitle**
See https://github.com/broadinstitute/workbench-libs/pull/270. No changes in Leo at this point besides updating the dependency.

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -12,7 +12,7 @@ object Dependencies {
 
   val workbenchUtilV    = "0.5-4c7acd5"
   val workbenchModelV   = "0.13-6dc016b"
-  val workbenchGoogleV  = "0.19-084fa1b"
+  val workbenchGoogleV  = "0.21-2b3109d-SNAP"
```
Comment (rtitle) [inline project/Dependencies.scala]: Update to non-SNAP once https://github.com/broadinstitute/workbench-libs/pull/270 is merged

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleComputeDAO.scala [hunk 2]
```diff
@@ -72,18 +73,18 @@ class HttpGoogleComputeDAO(appName: String,
   override def stopInstance(instanceKey: InstanceKey): Future[Unit] = {
     val request = compute.instances().stop(instanceKey.project.value, instanceKey.zone.value, instanceKey.name.value)
 
-    retryWhen500orGoogleError(() => executeGoogleRequest(request)).void.handleGoogleException(instanceKey)
+    retry(when5xx, whenUsageLimited, whenGlobalUsageLimited, when404, whenInvalidValueOnBucketCreation, whenNonHttpIOException)(() => executeGoogleRequest(request)).void.handleGoogleException(instanceKey)
```
Comment (kyuksel) [inline http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleComputeDAO.scala]: The above line is used so many times that I wonder if it makes sense to factor it out and give it a meaningful name if feasible.

**Review by kyuksel — APPROVED** [review]
👍 minor tinkering inline

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleComputeDAO.scala (hunk 2 above)
Comment (rtitle) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleComputeDAO.scala]: Yeah I agree - was just copy/pasting but that makes sense.

Comment (rtitle) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleComputeDAO.scala]: Done

**Pull request merged**